compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -212,22 +217,30 @@ getMatmulLoweringConfigAndWorkgroupSize(SmallVector<int64_t> bounds,
   for (int64_t mDim : contractionDims.m) {
     if (!ShapedType::isDynamic(bounds[mDim])) {
       mDims.push_back(mDim);
+    } else {
```

Comment:
nit: remove the negation in the `if` condition to simplify the logic? Currently it reads like double negation: support for unaligned is disabled when the shape is not dynamic...